services/application.go

```diff
@@ -138,6 +139,12 @@ func (app *ChainlinkApplication) AddJob(job models.JobSpec) error {
 	return app.JobSubscriber.AddJob(job, nil) // nil for latest
 }
 
+// ArchiveJob silences the job from the system, preventing future job runs.
+func (app *ChainlinkApplication) ArchiveJob(ID string) error {
+	_ = app.JobSubscriber.RemoveJob(ID)
```

Comment:
Wondering if we should still return an error here, possibly just `mutierr.Join` and in the happy path where they're both nil no change. I get that there's not much we can do if RemoveJob fails, but it would at least be good to surface the error in the case where it does turn up.

Reply:
Well, right now we attempt to remove all jobs from job subscriber, even those that aren't runlog/ethlog. We wouldn't want to log an error if the job was only `web`.